### ISSUE TITLE
Active focus in dialog tabs is not visible in the High Contrast mode

### ISSUE THREAD

**Issue by KarolDawidziuk** [ckeditor/ckeditor4#4934]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Switch your screen to High Contrast (HC) Mode.
2. Open https://ckeditor.com/docs/ckeditor4/latest/examples/fullpreset.html
3. Open dialog with tabs ie. Image
4. Move focus to one of the tabs.

### Expected result

The focus on the tab name is clearly visible.

### Actual result

Focus on the tab name does not occur. We can only change the active state.

https://user-images.githubusercontent.com/18471998/137901926-55b80bd5-8dcc-4b81-9cdb-b3920ae5907a.mov

## Other details

* Browser: Any
* OS: Windows

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5012
